packages/blade/src/components/ActionList/actionListUtils.ts

```diff
@@ -180,25 +182,29 @@ const validateActionListItemProps = ({
   trailing: ActionListItemProps['trailing'];
 }): void => {
   React.Children.map(trailing, (child) => {
```

Comment:
put the entire children.map under the dev check? why to even iterate if its not dev mode